test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -712,7 +716,7 @@
              in-request-handler (promise)
              ring-handler (fn [_]
                             (deliver in-request-handler true)
-                            (Thread/sleep 300)
+                            (Thread/sleep 2000)
```

Comment:
I think we could bump this up to something much higher, like 30 seconds, to be safer (when running in a super-slow Travis) and still get what we want out of this test.  For the cases that the shutdown happens right away (isn't graceful), this sleep should be interrupted anyway - so a much longer value shouldn't make the test take longer to run.  We could do something here that would be indefinite - like waiting for a promise that will never be delivered.  That might not be good though because in the event the test fails, we'd most likely get a hang.